src/data-and-backend/networking.md

```diff
@@ -27,7 +27,7 @@ manifest (`AndroidManifest.xml `):
 
 ### macOS
 
-macOS apps must allow network access in the relevant `.entitlements` files. 
+macOS apps must allow network access in the relevant `*.entitlements` files. 
```

Comment:
Should we be more specific with actual file names or link to more detail elsewhere?

Reply:
There's already a link to more details below. I think this is okay. I just threw me as I was looking for an `.entitlements` file in a hurry.